css/demo.css

```
@@ -0,0 +1,227 @@
+body {
+  display: grid;
+  grid-template: 'site-header' auto 'main' 1fr 'footer' auto / 100%;
+  margin: 0 var(--double-gap);
+  padding: var(--gap);
+}
+
+.site-header {
+  display: grid;
+  font-family: var(--sans-serif);
+  grid-area: site-header;
+  grid-template-columns: minmax(8ch, 15ch) auto;
+}
+
+#banner-logo {
+  &:link,
+  &:visited {
+    align-self: start;
+    display: block;
+    transition: transform 300ms cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    transform-origin: top center;
+    z-index: 1;
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    transform: scale(1.05);
+  }
+
+  [data-logo] {
+    block-size: auto;
+    inline-size: 100%;
+  }
+}
+
+[data-navlist] {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: var(--nav-placement, flex-end);
+  list-style: none;
+  margin-block: 0 var(--gap);
+  padding: 0;
+
+  @media (width >=40em) {
+    --nav-placement: start;
+  }
+}
+
+[data-nav] {
+  display: inline-block;
+  padding: 0 var(--shim);
+
+  &:not(:hover, :active, :focus, [aria-current]) {
+    --underline-color: transparent;
+    --underline-thickness: 0;
+  }
+}
+
+[data-layout='main'] {
+  grid-area: main;
+}
+
+.main-header {
+  container: main-header / inline-size;
+  position: relative;
+
+  @media (width >=60em) {
+    display: grid;
+    gap: var(--double-gap);
+    grid-template-columns: minmax(30ch, 1fr) minmax(20ch, 1fr);
+  }
```

@dvdherron we could change this to a flex container and increase the font-size max in the higher clamps and I think this gets us to where I think we want to be? All of my suggestions that I am leaving I think would work, but they may not all be necessary.... 
```suggestion
.main-header {
  container: main-header / inline-size;
  position: relative;
  justify-content: center;

  @media (width >=60em) {
    display: flex;
    gap: var(--double-gap);
  }
```